mobile/android/app/src/main/java/com/robosats/tor/TorKmpManager.kt

```diff
@@ -387,3 +388,16 @@ class TorKmpManager(application : Application) {
         }
     }
 }
+
+object TorKmpManager {
+    private lateinit var torKmp: TorKmp
+
+    @Throws(UninitializedPropertyAccessException::class)
+    fun getTorKmpObject(): TorKmp {
+        return torKmp
+    }
+
+    fun updateTorKmpObject(newKmpObject: TorKmp) {
+        torKmp = newKmpObject
+    }
+}
```

Comment:
You can´t have 2 TorKmp running on your app, so this singleton makes sure it's accessible from the app and the notification foreground service